guides/v2.3/graphql/develop/get-customer-authorization-token.md

```diff
@@ -0,0 +1,39 @@
+---
+group: graphql
+title: How to Get Customer Authorization Token
+---
+
+There are GraphQl queries that can not be successfully executed without customer authentication. GraphQl realizes customer authentication through 
```

Comment:
Here's a suggested rewrite of this topic pending your feedback:
-----------
To successfully retrieve information about a specific customer using GraphQL, you need to provide the customer's authentication token. 

The following example shows how to retrieve a customer's authorization token:

**Request**

```text
mutation {
  generateCustomerToken(email: "customer@example.com", password: "password") {
    token
  }
}
```
**Response**

```json
 {
   "data": {
     "generateCustomerToken": {
       "token": "c9l4ihcgs45kz2c8h7bb2jp4mzyzz7q0"
     }
   }
 }
```
You can now use this token in the `Authorization` request header field for any queries and mutations.

![GraphiQL Authorization Bearer]({{ page.baseurl }}/graphql/images/graphql-authorization.png)

Reply:
**implemented** 

Thank you, @erikmarr 👍 

P.S. Also I've added few words hot to revoke token. Could you please review? 😊 

Thank you very much!

Reply:
Rather than adding information about how to revoke a customer's token, can you instead link to the following topic that already contains this information?
https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html

Reply:
> Rather than adding information about how to revoke a customer's token, can you instead link to the following topic that already contains this information?
> https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html

**done**

Could you please review now?

Thank you, @erikmarr !